Merging Weapons

JMx36 opened the pull request.

alegottu committed:
media controller skeleton for weapons added

zhaoKelvin committed:
basic animations

JMx36 committed:
Prop Manager and Animation

JMx36 committed:
deleted prop and prop manager

JMx36 committed:
deleted Manager folder

Reallife101 committed:
Add managers back in

Reallife101 committed:
Merge pull request #9 from Reallife101/main

Resetting

JMx36 committed:
Merge pull request #10 from Reallife101/kelvin+alex

Merging Weapons into Josh

Reallife101 commented:
Comments said in person: add weapons to diana's scene

Reallife101 commented:
Test

alegottu committed:
Merging with main to integrate weapons with health

alegottu committed:
integrated weapons with health system

JMx36 committed:
Started implementing level scene

JMx36 committed:
Merge branch 'Josh' into kelvin+alex

JMx36 committed:
Merge pull request #12 from Reallife101/kelvin+alex

pull latest weapon changes

nedlemethm committed:
Movement Rework #1

Adjusted the gravity on the player and made changes to allow either hold to sprint or toggle sprint

alegottu committed:
integrated basic animations

alegottu committed:
fixing scene merge conflict between kelvin and my version

alegottu committed:
added puddle effect and weapon animations

Reallife101 reviewed (changes requested):
Currently, it swaps out mesh renderers for the clean and dirty. However, we sometimes have different meshes and models completely (ex: a stack of messy books vs a stack of neatly stacked books) which means they could have different conditions. As a result, can we change these to swap gameObjects instead?

As for the current pull request on Github, there is an issue that when I shoot 1 box, it effects all the boxes (I made sure to remove the bit that makes them take damage when I clicked). Also, the player should be tagged with the player tag in general, helps with various collisions stuff such as ending the level from diana's branch.

Because of these things, I currently can't accept the merge request for Merging weapons.

JMx36 committed:
Fixed Enemy Bug: All dying at the same time

JMx36 committed:
Forgot to save these changes

alegottu committed:
Merge remote-tracking branch 'origin/Leo_MovementWIP' into kelvin+alex

alegottu committed:
final changes for MVP

coleyoung02 committed:
timer back in

I also put in the fancier health bar but im not going to make it functional until the last minute

JMx36 committed:
Added an extra enemy to my scene and fix a mesh property from an enemy

JMx36 committed:
Merge pull request #17 from Reallife101/cole-2

Adding timer to Josh

JMx36 committed:
timer merged. Prints out your time when level is done.

JMx36 committed:
Merge pull request #18 from Reallife101/kelvin+alex

Latest weapons and movements changes

JMx36 committed:
Update on Josh's Scene

JMx36 committed:
Added another timer script

Reallife101 committed:
Try to fix timer

JMx36 committed:
moved other_time script
